docs/guide/search/elastic.md

```diff
@@ -0,0 +1,558 @@
+
+    #### Fine-tune the search results
+
+    Your search results can be adjusted by configuring additional parameters.
+    For a list of available mapping parameters and their usage, see [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.x/mapping-params.html).
+
+    For example, you can apply a mapping parameter, in this case, a normalizer, to a specific
+    mapping under the `dynamic_templates` key:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    # ...
+                    dynamic_templates:
+                        - ez_string:
+                            match: "*_s"
+                            mapping:
+                            type: keyword
+                            normalizer: lowercase_normalizer
+                        # ...
+    ```
+
+    You can also set a boosting factor for a specific field.
+    Boosting increases the relevance of hits, for example making keywords from the title more relevant than the ones from other places of the document.
+    Set the boosting factor under the `properties` key:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    properties:
+                        content_name_s:
+                            boost: 2.0
+                    # ...
+    ```
+
+    You can even copy contents of existing fields, process them and then paste into another field, which than can be queried:
+
+    ``` yaml
+    ez_platform_elastic_search_engine:
+        # ...
+        index_templates:
+            default:
+                # ...
+                mappings:
+                    properties:
+                        user_first_name_s:
+                            type: keyword
+                            normalizer: lowercase_normalizer
+                            copy_to: custom_field
+                    # ...
+    ```
+
+    ### Bind templates with connections
+
+    Once you have created the field mapping template(s), you must establish a relationship between the templates and a connection. You do this by adding the "index_templates" key to a connection definition.
+
+    If your configuration file contains several connection definitions, you can reuse the same template for different connections.
+    If you have several index templates, you can apply different combinations of templates to different connections.
+
+    ``` yaml
+    <connection_name>:
+        # ...
+        index_templates:
+            - default
+            - default_en_us
+    ```
+
+    ## Step 5: Push the templates
+
+    Push the templates to the Elasticsearch engine by running the following command:
```

Comment:
We should make it clear that this step should be done for each of the defined connection.